trade-finance-manager-ui/component-tests/utilisation-reports/_macros/payment-details-table-row.component-test.ts

```diff
@@ -222,7 +206,10 @@ describe(component, () => {
           },
         },
         reconciledBy: 'Some reconciled by user',
-        dateReconciled: 'Some reconciled date',
+        dateReconciled: {
+          formattedDateReconciled: 'Some reconciled date',
```

Comment:
Should this not be a date? For realistic mocking, same on line 163

Reply:
Sure. For reference, the reason we don't try to enforce realistic data in component tests is that your components should be fairly indifferent to the kind of data you supply (outside of the shape of the object)